docker-compose.yml

```diff
@@ -0,0 +1,13 @@
+version: '3.9'
+services:
+    web:
+        build: .
```

Comment:
Again, it would be nicer to specify a target stage rather than rely on the ordering in the Dockerfile.